src/status_im/contexts/wallet/common/temp.cljs

```diff
@@ -1,43 +0,0 @@
-(ns status-im.contexts.wallet.common.temp
-  (:require
-    [react-native.core :as rn]
-    [status-im.common.resources :as status.resources]
-    [utils.i18n :as i18n]))
-
-(def buy-tokens-list
-  [{:title             "Ramp"
-    :description       :text
-    :description-props {:text (i18n/label :t/ramp-description)}
-    :tag               :context
-    :tag-props         {:icon    :i/fees
-                        :context "0.49% - 2.9%"}
-    :action            :arrow
-    :action-props      {:alignment :flex-start
-                        :icon      :i/external}
-    :image             :icon-avatar
-    :image-props       {:icon (status.resources/get-service-image :ramp)}
-    :on-press          #(rn/open-url "https://ramp.com")}
-   {:title             "MoonPay"
-    :description       :text
-    :description-props {:text (i18n/label :t/moonpay-description)}
-    :tag               :context
-    :tag-props         {:icon    :i/fees
-                        :context "1% - 4.5%"}
-    :action            :arrow
-    :action-props      {:alignment :flex-start
-                        :icon      :i/external}
-    :image             :icon-avatar
-    :image-props       {:icon (status.resources/get-service-image :moonpay)}
-    :on-press          #(rn/open-url "https://moonpay.com")}
-   {:title             "Latamex"
-    :description       :text
-    :description-props {:text (i18n/label :t/latamex-description)}
-    :tag               :context
-    :tag-props         {:icon    :i/fees
-                        :context "1% - 1.7%"}
-    :action            :arrow
-    :action-props      {:alignment :flex-start
-                        :icon      :i/external}
-    :image             :icon-avatar
-    :image-props       {:icon (status.resources/get-service-image :latamex)}
```

Comment:
@ajayesivan  - If it's not too much trouble, would you mind deleting the local images/assets for providers as we are using the URL to fetch images? This would save some space in the app.

Reply:
Sure